packages/wrangler/src/index.tsx

```diff
@@ -501,6 +511,48 @@ export function createCLIParser(argv: string[]) {
 		}
 	);
 
+	wrangler.command(
```

Comment:
can you extract this command into its own module?

i.e create `deployments/index.ts` so the command can be like KV's above (if there are plans to add subcommands):
```ts
	wrangler.command(
		"kv:bulk",
		"💪 Interact with multiple Workers KV key-value pairs at once",
		(bulkYargs) => {
			return kvBulk(bulkYargs.command(subHelp));
		}
	);
```

Reply:
We can abstract when we actually add more features or subcommands. 😄